design/carvel-install-design.md

```diff
@@ -0,0 +1,24 @@
+# Design proposal for Velero Installation with Carvel
+
+Currently, the Velero CLI tool has a `install` command that configures numerous major and minor aspects of Velero. This document outlines how we can also have Velero be installed and upgraded using the Carvel tooling.
+
+
+## Goals
+- Have one source of truth for all Velero configuration
+- Have compatibility with gitop practices (i.e. ability to generate a full set of yaml for install that can be stored in source control)
```

Comment:
Should we add a requirement to output yaml as a "dry run" to handle this?  Will Carvel be able to do this or would we need to do this as part of the velero install command, calling a Carvel library?

Reply:
I will add ithis as a requirement to be sure. Carvel has this functionality, and so does the Velero install, this should definetely continue to be available one way or another.